### ISSUE TITLE
[Request] pulsar-bin

### ISSUE THREAD

**Issue by tio-trom** [chaotic-aur/packages#2267]
### Link to the package base(s) in the AUR

https://aur.archlinux.org/packages/pulsar-bin

### Utility this package has for you

A community-led hyper-hackable text editor, built on electron

### Do you consider the package(s) to be useful for every Chaotic-AUR user?

No, but for a great amount.

### Do you consider the package to be useful for feature testing/preview?

- [ ] Yes

### Have you tested if the package builds in a clean chroot?

- [X] Yes

### Does the package's license allow redistributing it?

YES!

### Have you searched the issues to ensure this request is unique?

- [X] YES!

### Have you read the README to ensure this package is not banned?

- [X] YES!

**Comment by Technetium1** [comment]
This is a rather large package..

**Comment by tio-trom** [comment]
> This is a rather large package..

How large is it?

**Comment by Technetium1** [comment]
~200mb, just for a deb file that gets extracted, not sure if this is worth it.

**Comment by tio-trom** [comment]
Ah thats a lot. Yeah I totally understand. And since it already has a bin package we can skip this ;).

**Comment by dr460nf1r3** [comment]
I agree, especially since we are trying to not increase space used by a lot.
